### DISCUSSION TITLE
How to re-process leftover files in /upload (Storage template: On)

### DISCUSSION THREAD

**Discussion by chengguangnan** [immich-app/immich#11607] [Q&A] [answered]
I found some files under /upload. Is there a web UI job to pick them up? Or should I upload them again with command line?

**Comment by chengguangnan** [comment]
I copied them into a folder to review, `time fd -t f -x cp {} check/`, most large .mov files are corrupted, but some .heic file looks fine.

**Comment by mertalev** [comment, marked as answer]
If Immich didn't import these files, it generally means something went wrong and the files are partial.

I recommend searching in Immich based on the date metadata to see if these assets got uploaded some other time - they would exist as different assets independent from the files you're seeing.

If they don't exist, the best option is to find the originals and upload those.

You *can* import the files in the folder, but I don't really recommend it because of the risk of corruption.

**Reply by chengguangnan** [reply]
Thank you!
